main_node:
  ros__parameters:
    # Robot parameters
    robot_name: "jackal_ugv"
    max_lin_vel: 0.6
    min_lin_vel: 0.05
    max_ang_vel: 1.5
    min_ang_vel: 0.05
    robot_type: "rectangular" # rectangular or circular
    robot_radius: 0.2 # [m], If robot_type is circular
    robot_size: [0.508, 0.430] # [m, m], Lenght, Width if robot_type is rectangular
    collision_tolerance: 0.05 # [m], tollerance of collision added to robot_radius or robot_size
    warning_tolerance: 0.1 # [m], warning window added to robot_radius or robot_size
    goal_tolerance: 0.40

    # World parameters
    world_name: "social_indoor.world" #Leave empty to obtain no world
    data_path: "social_indoor.json"
    agents_config: "social_indoor_agents.yaml"

    # Task parameters
    task: 'goToPose' # goToPose, Following, Vineyards
    
    # Sensors parameters
    sensor: 'applr' # lidar or rgb_camera or depth_camera
    applr_param:
      update_frequency: 10.0 # update the controller parameters at this frequency
    visual_data: 'features'
    features: 12544
    channels: 1
    imu_enabled: "false"
    camera_enabled: "false"
    stereo_camera_enabled: "false"
    lidar_enabled: "true"
    sensors_topic:
      rgb_topic: "/camera/image_raw"
      depth_topic: "/camera/depth/image_raw"
      laser_topic: "/scan"
      imu_topic: "/imu/data"
      odom_topic: "/odom"
    laser_param:
      max_distance: 3.0 # Meters
      num_points: 36 # int divider of 360
    depth_param:
      dist_cutoff: 5.0 # Meters
      width: 112
      height: 112
      show_image: False
    rgb_param:
      width: 640
      height: 480
      show_image: False 
